src/Numerics/LinearAlgebra/Vector.Arithmetic.cs

```diff
@@ -957,53 +975,440 @@ public void PointwiseRemainder(Vector<T> divisor, Vector<T> result)
         }
 
         /// <summary>
-        /// Pointwise applies the exponent function to each value.
+        /// Takes a function and 
         /// </summary>
-        public Vector<T> PointwiseExp()
+        /// <param name="f"></param>
+        /// <returns></returns>
+        protected Vector<T> PointwiseUnary(Action<Vector<T>> f)
         {
             var result = Build.SameAs(this);
-            DoPointwiseExp(result);
+            f(result);
             return result;
         }
 
         /// <summary>
-        /// Pointwise applies the exponent function to each value.
+        /// 
         /// </summary>
-        /// <param name="result">The vector to store the result.</param>
-        /// <exception cref="ArgumentException">If this vector and <paramref name="result"/> are not the same size.</exception>
-        public void PointwiseExp(Vector<T> result)
+        /// <param name="f"></param>
+        /// <param name="result"></param>
+        protected void PointwiseUnary(Action<Vector<T>> f, Vector<T> result)
```

Comment:
Please avoid adding empty documentation skeletons, that makes it hard to spot whether we're still lacking some proper documentation.